src/Components/Header.tsx

```diff
@@ -1,54 +1,60 @@
 import React from 'react';
-import { translate } from 'react-translate';
-
-import Link from '../Components/LowerCaseUrlLink';
+import Link from './LowerCaseUrlLink';
 import { englishCode, spanishCode } from '../Utilities/LanguageCodes';
 import MessageBox from './MessageBox';
+import { ParsedQs } from 'qs';
+import { useIntl } from 'react-intl';
+
+interface HeaderProps {
+  message?: string | ParsedQs | string[] | ParsedQs[] | undefined,
+  changeLanguage: (newLanguage: any, newUrl?: any) => void,
```

Comment:
The definition can be re-used from LocalizedApp.SetLanguageType

Reply:
[9ede5d4](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/9ede5d4e8127366824a28d497840cf0cf5e1cc2e)